<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>531</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loading_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="connectTip_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>正在连接服务器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="loadingImage_lab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="intoMainWindow_btn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(107, 107, 107);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>进入系统</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainWindow_page">
     <widget class="QLabel" name="COG_lab">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>COG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="SOG_lab">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>SOG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="COG_num_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>256.7°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="SOG_num_lab">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>32.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="location_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>位置：</string>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>211</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="latitude_lab">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>纬度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="longitude_lab">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>经度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="latitude_str_lab">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>35°45.21′ N</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="longitude_str_lab">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>113°26.52′ E</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="wind_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>风况：</string>
      </property>
     </widget>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="realWindSpeed_lab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>真风速：</string>
      </property>
     </widget>
     <widget class="QLabel" name="wind_direction_lab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>400</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>风向：</string>
      </property>
     </widget>
     <widget class="QLabel" name="wind_speed_lab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>440</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>速度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="kn_lab">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>kn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="wind_direction_num_lab">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>400</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>39.1°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="wind_speed_num_lab">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>4.7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="kn_lab_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>440</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>kn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="water_depth_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>水深：</string>
      </property>
     </widget>
     <widget class="QLabel" name="water_depth_num_lab">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>480</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>32.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="kn_lab_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(154, 154, 154);</string>
      </property>
      <property name="text">
       <string>kn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
